migrations/2022-11-20-032430_sticky_local/up.sql

```diff
@@ -0,0 +1,42 @@
+
+DROP TRIGGER IF EXISTS post_aggregates_stickied ON post;
+drop function 
+  post_aggregates_stickied;
+
+
+alter table post ADD featured_community boolean NOT NULL DEFAULT false;
```

Comment:
This is really just a rename, so do that instead of creating a new column, copying values, then removing the column.

Reply:
Sorry, i did the new column so that the schema has the two featured columns together in order, rather than far apart. Looks cleaner. Is that alright?

Reply:
That's fine I spose.